composer.json

```diff
@@ -19,39 +19,41 @@
         "erusev/parsedown": "^1.6",
         "sensio/framework-extra-bundle": "^6.1",
         "symfony/apache-pack": "^1.0",
-        "symfony/asset": "^5.3",
-        "symfony/console": "^5.3",
-        "symfony/dotenv": "^5.3",
-        "symfony/expression-language": "^5.3",
+        "symfony/asset": "5.4.0-beta1",
```

Comment:
should probably use `^5.4.0-beta1` so that it can update to newer beta releases on next `composer update`, and then to stable releases.

Reply:
Yes. Done!